packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -65,6 +65,10 @@ export default mixins(
       type: [String, Number],
       default: 0,
     },
+    localeFirstDayOfYear: {
```

Comment:
Order alpha.

Reply:
Do you mean that the props should be ordered in alpabetical order?

Reply:
Ordering in alpha order done